lib/workers/repository/index.ts

```diff
@@ -23,7 +24,9 @@ export async function renovateRepository(
   canRetry = true
 ): Promise<ProcessResult> {
   splitInit();
-  let config = GlobalConfig.set(repoConfig);
+  let config = GlobalConfig.set(
+    applySecretsToConfig(repoConfig, undefined, false)
```

Comment:
```suggestion
    applySecretsToConfig(repoConfig, {}, false)
```
I think we need to pass an empty object to be safe for strict null checks